conduct.md

```diff
@@ -63,35 +66,49 @@ Allegheny's honor code for all submitted work.
 * Acceptable Conduct (A):
   * Members shall
     * Watch the course website and Slack workspace for changes to due dates.
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+    * Turn assignments in on time.
+    * Complete their work in a timely manner that will not negatively affect other group members.
 
 * Unacceptable Conduct (U):
   * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for deadlines.
+    * Be dishonest about late assignments.
+    * Bargain for more time to complete assignments.
 
 #### Communication
 
 This section defines what communication is acceptable and unacceptable. It is vital that team members not only communicate clearly, but communicate using the right platforms and methods, such as GitHub issues tracker or slack. For example, something may be clearly communicated in Slack, but may still receive an 'unacceptable' because Slack was the inappropriate platform to use.
 
 * Acceptable Conduct (A):
   * Members shall
-    * Use appropriate softwares for electronic communication.
+    * Use appropriate softwares for electronic communication, such as Slack or GitHub.
     * Always participate in the discussion.
+    * Keep all communication respectful.
+    * Regularly communicate questions and issues with the class.
+    * Avoid speaking over other people.
+    * Give and accept constructive criticism.
     * Listen to and consider ideas presented by others.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for communication.
 
 * Unacceptable Conduct (U):
   * Members shall not
     * Frequently use inappropriate forums such as text messaging or using Slack instead of GitHub issues tracker.
     * Rarely engage in the discussion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for communication.
+    * Use communication platforms inappropriately.
+    * Attend lab sessions and meetings without appropriate electronic communication devices.
 
 ## Team Work
 
 Some of the class activities and lab projects will require team collaboration.
 This section of the Code of Conduct will provide guidance on effective and
 inclusive team work.
 
+* Acceptable Conduct (A):
+  * Members shall
+    * Work together as a group with the work split up tp complete the assignment.
+
+* Unacceptable Conduct (U):
+  * Members shall not
+    * Put all the work on one or two people in the group to do.
+
```

Comment:
I think that `Teamwork` is a level above `Participation`, `Decisions`, etc.. So the text under it describes this whole section related to teamwork. You can either remove this or incorporate parts of it in the other `Team Work` sub-sections.